src/Main.ts

```diff
@@ -1120,6 +1120,74 @@ export class Main {
         }
     }
 
+    public async getChannelInfo(opts: {
+        slack_channel_id: string,
+        slack_bot_token: string,
+        team_id: string,
+    }): Promise<ConversationsInfoResponse|void> {
+        let slackClient: WebClient | undefined;
+        let teamEntry: TeamEntry | null = null;
+        let teamId: string = opts.team_id;
+
+        const existingChannel = opts.slack_channel_id ? this.rooms.getBySlackChannelId(opts.slack_channel_id) : null;
+
+        if (existingChannel) {
+            throw Error("Channel is already bridged! Unbridge the channel first.");
+        }
+
+        if (opts.slack_bot_token) {
```

Comment:
We shouldn't be passing the slack token in here

Reply:
Removed everything related to botToken for getChannelInfo.